spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/trace/pubsub/TracePubSubAutoConfiguration.java

```diff
@@ -50,4 +55,14 @@ static TracePubSubBeanPostProcessor tracePubSubBeanPostProcessor(BeanFactory bea
   PubSubTracing pubSubTracing(MessagingTracing messagingTracing) {
     return PubSubTracing.newBuilder(messagingTracing).build();
   }
+
+  @Bean
+  @ConditionalOnMissingBean
+  PublisherCustomizer publisherCustomizer(PubSubTracing pubSubTracing) {
```

Comment:
This is specifically the `tracePublisherCustomizer`. So, I think it needs to be named that way.